Patch 6 - Cosmetic update

dwesely opened the pull request:
Reorganized the page to fit without scrolling, moved Create/Parse buttons over to the left, reordered the settings controls to be horizontal, turned off wrapping of the text areas so wide tables don't wrap, centered the generated html table.

I wanted to use the Bootstrap lessons I just learned in the free code camp :-)

dwesely committed:
Cosmetic update

Reorganized the page to fit without scrolling, moved Create/Parse buttons over to the left, reordered the settings controls to be horizontal, turned off wrapping of the text areas so wide tables don't wrap.

I wanted to use the Bootstrap lessons I just learned in the free code camp :-)

dwesely committed:
Cosmetic update

Removed unused styles, added scrollbars to text areas, text-decoration: none for the pre tag to keep underlines off in internet explorer.

dwesely committed:
Cosmetic update

Center the generated html table so it matches the formatting of the rest of the page.

ozh commented:
I think several things don't work here
- footer doesn't stick to the page bottom (or was it already the case and it just shows now?)
- lots of scrollbars everywhere, even when not needed
- the page itself gets an horizontal scrollbar (FF at least)
- on wide screen size, controls are way too wide
- on narrower screen size, there's a scrollbar mayhem

Overall I think it loses much of its "simple" look and feel

dwesely committed:
Scaling width

As the page gets bigger or smaller, the buttons shift so they don't look too big with a big display or too small with a small display

dwesely committed:
-Scrollbars and corrected footer

Removed permanent scrollbars from the textareas, put the footer back at the foot

dwesely commented:
- footer doesn't stick to the page bottom (or was it already the case and it just shows now?)
  - Footer now located at the bottom of the window, or the bottom of the content, whichever is lower
- lots of scrollbars everywhere, even when not needed
  - Removed perma-scrollbars
- the page itself gets an horizontal scrollbar (FF at least)
  - Fixed the div width issue that was causing horizontal scrollbar
- on wide screen size, controls are way too wide
  - Used [bootstrap column grids](http://getbootstrap.com/css/#grid) to scale the size of controls with the screen size
- on narrower screen size, there's a scrollbar mayhem
  - Reduced mayhem

ozh commented:
Nice :)
